kt/godot-runtime/src/main/kotlin/godot/runtime/Registration.kt

```diff
@@ -31,7 +31,8 @@ data class KtFunctionArgument(
 
 class ClassBuilderDsl<T : KtObject>(
     @PublishedApi internal val name: String,
-    private val superClass: String
+    private val superClass: String,
```

Comment:
Don't know if we should keep it. It is used only for logging.

Reply:
I would remove it if it's not needed.
The entry generation really is only working for kotlin, so if one wants to use another language, he has to write the entry file manually.  So the lesser one has to write, the better IMO

Reply:
Should we now leave it or remove it?

Reply:
I'll remove it today :)

Reply:
But the entry generator will also need updating then.
Do you do it or should i?

Reply:
you can do entry gen, I'll adapt cpp code after if you want :)

Reply:
To sum up discord discussion, we'll keep `KtClass::super_class` for now and see what we do with `KotlinScript::get_base_script`.